components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html

```diff
@@ -1,80 +1,98 @@
 {% extends "layout.html" %}
 {% block javascript %}
-  {{ super() }}
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js"></script>
-  <script src="{{ prefix + url_for('static', filename='js/notebooks.js') }}"></script>
+	{{ super() }}
+    <script defer>
+        // Establish communication with Central Dashboard
+        window.addEventListener('DOMContentLoaded', function (event) {
+            const {CentralDashboardEventHandler: CD_EvHandler} = window.centraldashboard || {}
+            if (!CD_EvHandler) return
+            CD_EvHandler.init(function(handler, isIframed) {
+                document.body.classList[isIframed?'add':'remove']('iframe-view')
+                if (!isIframed) return
```

Comment:
Let's try to adhere to our normal JS style guidelines from the Google JS style guide with respect to semi-colons and one variable declaration per line.

https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration.